ixp_tracker/stats.py

```diff
@@ -77,20 +81,23 @@ def generate_stats(geo_lookup: ASNGeoLookup, stats_date: datetime = None):
         country_stats = all_stats_per_country[code]
         if country_stats.get("all_asns") is None:
             country_stats["all_asns"] = geo_lookup.get_asns_for_country(code, stats_date)
+        if country_stats.get("routed_asns") is None:
+            country_stats["routed_asns"] = geo_lookup.get_routed_asns_for_country(code, stats_date)
         local_asns_members_rate = calculate_local_asns_members_rate(country_stats["member_asns"], country_stats["all_asns"])
```

Comment:
question: oh. We might need to rethink this. I thought the whole point of this change in IXP Tracker was to use active ASes for this metric (as that's the metric exposed on CR that raised the whole registered vs active difference). I hadn't clocked before but of course it probably doesn't make sense to use active ASes here while we are using registered status for membership. Perhaps one for the wider team? (FWIW I don't think the original work to pull the active ASes was wasted, that feels like a worthwhile change)